sqs/Module.md

```diff
@@ -1,83 +1,25 @@
-
-**Client Operations**
-Connector contains operations that create and delete queue, send messages, receive messages and delete messages in a queue.
-
-
-![image](docs/images/aws_sqs_connector.png)
-
-## Compatibility
-| Ballerina Language Versions | Amazon SQS API version  |
-| --------------------------- | ----------------------  |
-|     Swan Lake Alpha 5       |        2012-11-05       |
-
-The following sections provide you with information on how to use the Ballerina Amazon SQS connector.
-
-### Contribute to development
-
-Clone the repository by running the following command 
-```shell
-git clone git@github.com:ballerina-platform/module-ballerinax-aws.sqs.git
-```
-
-### Signing Up for AWS
-
-1. Navigate to [Amazon](https://aws.amazon.com), and then click `Create an AWS Account`.  
-
-    **Note**: If you previously signed in to the AWS Management Console using the root user credentials of the AWS account, click `Sign in` to use a different account. If you previously signed in to the console using the IAM credentials, sign in using the credentials of the root account.
-
-2. Click `Create a new AWS account` and follow the given instructions.  
-
-Follow the method explained below to obtain AWS credentials.
-
-### Obtaining Access Key ID and Secret Access Key to Run the Sample
-
-1. Sign in to the AWS Management Console and open the IAM console at https://console.aws.amazon.com/iam/.
-2. In the navigation pane, choose `Users`.
-3. Choose the name of the user whose access keys you want to create, and then choose the `Security credentials` tab.
-4. In the `Access keys` section, choose `Create access key`.
-5. To view the new access key pair, choose `Show`. You will not have access to the secret access key again after this dialog box closes. Your credentials will look something like this:
-    - Access key ID: AKIAIOSFODNN7EXAMPLE
-    - Secret access key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY  
-
-For more information please visit https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-setting-up.html .  
-
-# Running Tests
-1. Create Config.toml file in module-ballerinax-aws.s3 with the following configurations and provide appropriate value.
-```ballerina
-accessKeyId = "testAccessKeyValue"
-secretAccessKey = "testSecretAccessKeyValue"
-region = "testRegion"
-accountNumber = "testAccountNumber"
-```
-2. Navigate to the `module-ballerinax-aws.sqs` directory.
-3. Run tests :
-```ballerina
-bal test
-```
-
-# Quickstart
-
-## Import the AWS SQS module
+### Step 1: Import connector
 First, import the ballerinax/aws.sqs module into the Ballerina project.
-
 ```ballerina
 import ballerinax/aws.sqs;
 ```
-
-## Initialize the AWS SQS Client giving necessary credentials
+### Step 2: Create a new connector instance
 
 You can now enter the credentials in the SQS client configuration and create SQS client by passing the configuration:
```

Comment:
Suggestion: "You can now enter the credentials in the SQS client configuration and create the SQS client by passing the configuration as follows."

Reply:
Fixed